pkg/contracts/solana/gateway_message.go

```diff
@@ -231,11 +232,14 @@ type MsgExecute struct {
 	signature [65]byte
 
 	// Sender is the sender address for the execute
-	sender [20]byte
+	sender string
 
 	// Data for execute
 	data []byte
 
+	// revert marks if it's execute on execute_revert
```

Comment:
Could we rather have something like a enum for execute type? 
Something like `CALL | REVERT`